tests/testthat/test-histogram.R

```diff
@@ -31,19 +31,13 @@ test_that('expect stability mechanism for unknown variable type', {
     
     expect_error(dpHistogram$new(varType='number data', variable="educ", n=nTest, epsilon=epsilonTest, 
                                  nBins=nBinsTest, delta=deltaTest, rng=c(0,16)), 
-                 "Please enter a data type of 'numeric', 'integer', 'logical', or 'character'")
+                 "Variable type number data should be one of numeric, integer, logical, character", fixed=TRUE)
```

Comment:
shouldn't the value passed into varType be passed into the error message? Right now the error message seems to be hard coded to have the current value of varType.

Reply:
the error message is hardcoded only in the sense that the expected error in this case is hardcoded in the testthat function. But in general, checkVarType does customize the error message; on a different varType or in a different stat it will output differently.